docker/jupyter/scripts/run-jupyter.sh

```diff
@@ -5,9 +5,11 @@ set -e
 # Change the default umask to support R/W access to the shared volume with welder
 umask 002
 
+NOTEBOOKS_DIR=$1
```

Comment:
[This](https://github.com/DataBiosphere/leonardo/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala#L134) also needs to be updated if `run-jupyter.sh` now takes an arg.

Reply:
Also it might be nice if this had a default:
```
NOTEBOOKS_DIR=${1:-${HOME}/notebooks}
```
What do you think?